core-js/src/test/javascript/cdf/dashboard/Utils-spec.js

```diff
@@ -116,5 +116,27 @@ define(["cdf/dashboard/Utils", "cdf/lib/CCC/cdo"], function(Utils, cdo) {
       expect(Utils.ev("1")).toBe("1");
       expect(Utils.ev(function() { return "1"; })).toBe("1");
     });
+
+    /**
+     * ## The Utils class # verify if pathIncludesAction evaluates well
+     */
+    it( "evaluate if path includes action", function () {
+      expect( Utils.pathIncludesAction( "a.ext", "a.ext" ) ).toBe( false );
```

Comment:
I disagree with this one. Already explained above.

Reply:
Same above